### PR TITLE
Parent home

### PR DESCRIPTION
**Pull request by TorusTredent**

### CODE REVIEW

File: accounts/src/main/java/com/jungeeks/controller/ExController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class ExController extends ResponseEntityExceptionHandler {
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/controller/ExController.java]: Rename

File: accounts/src/main/java/com/jungeeks/dto/ChildDto.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+
+@Data
+@AllArgsConstructor
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/dto/ChildDto.java]: What type of constructor do you need? There are 3 types now

File: accounts/src/main/java/com/jungeeks/dto/ParentHomeDto.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/dto/ParentHomeDto.java]: Same Q as in ChildDto

File: accounts/src/main/java/com/jungeeks/controller/ExController.java (hunk 1 above)
Comment (TorusTredent) [reply accounts/src/main/java/com/jungeeks/controller/ExController.java]: fix

File: accounts/src/main/java/com/jungeeks/dto/ChildDto.java (hunk 2 above)
Comment (TorusTredent) [reply accounts/src/main/java/com/jungeeks/dto/ChildDto.java]: fix

File: accounts/src/main/java/com/jungeeks/dto/ParentHomeDto.java (hunk 3 above)
Comment (TorusTredent) [reply accounts/src/main/java/com/jungeeks/dto/ParentHomeDto.java]: fix

**Review by VolgaDaniluk — APPROVED** [review]

**Pull request merged**